### DISCUSSION TITLE
The discussion board

### DISCUSSION THREAD

**Discussion by mestiez** [studio-minus/ppg-bugs-and-requests#2010] [Announcements]
# Hello to the 5 people that read this

Instead of having extended off-topic discussions inside the issue tracker, you can now have them here. Occasionally I will post some announcements here too in the hopes of sparking discussions that are less unhinged than what would happen on Discord or Reddit.

**Comment by royalealbert27** [comment]
unfortunately, your mother

**Comment by Drogo-man** [comment]
*in french accent* off to visit you mother!

**Comment by thetruss** [comment]
thetruss
